### ISSUE TITLE
[Plugin][Enh] ContainerTask error message

### ISSUE THREAD

**Issue by JusteRaimbault** [openmole/openmole#517]
As illustrated in #516 , when a `ContainerTask` fails, the error message is not explicit on what happened inside the container.
The user does not necessarily think of checking container log - could we retrieve this output and provide a more explicit error message in that case?

**Comment by romainreuillon** [comment]
We could add: "The process might have written some errors on the standard output. You may want to check the log of the standard output for more information on this error."

Would the wording be okay?

**Comment by romainreuillon** [comment]
throw new UserBadDataError(s"Process \"$commandValue\" exited with an error code $retCode (it should equal 0). The process might have written some errors on the standard output. You may want to check the log of the standard output for more information on this error.")

**Comment by JusteRaimbault** [comment]
Yes I think this would be a good addition!
Also, is it totally impossible to retrieve the container log and to provide its last lines here? or this wouldn't make sense as too many different situations can occur?

**Comment by romainreuillon** [comment]
Okay, I implemented in : 7108d8e917cca566b06cdbcc84c9d772e40eb4db

**Comment by romainreuillon** [comment]
Looks like:
```
	at org.openmole.core.workflow.execution.LocalExecutor.run(LocalExecutor.scala:55)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.openmole.core.exception.UserBadDataError: Process "echo $((9*2)) ; exit 1" exited with an error code 1 (it should equal 0).
The last lines of the standard output were:
|18
You may want to check the log of the standard outputs for more information on this error.
```
